src/main.cpp

```diff
@@ -123,30 +138,39 @@ void wrongDataState(){
 }
 
 /***
- * Avarage methods
+ * Average methods
  **/
 
-void saveDataForAvarage(unsigned int pm25, unsigned int pm10){
+void saveDataForAverage(unsigned int pm25, unsigned int pm10){
   v25.push_back(pm25);
   v10.push_back(pm10);
 }
 
-unsigned int getPM25Avarage(){
-  unsigned int pm25_avarage = accumulate( v25.begin(), v25.end(), 0.0)/v25.size();
+unsigned int getPM25Average(){
+  unsigned int pm25_average = accumulate( v25.begin(), v25.end(), 0.0)/v25.size();
   v25.clear();
-  return pm25_avarage; 
+  return pm25_average;
 }
 
-unsigned int getPM10Avarage(){
-  unsigned int pm10_avarage = accumulate( v10.begin(), v10.end(), 0.0)/v10.size();
+unsigned int getPM10Average(){
+  unsigned int pm10_average = accumulate( v10.begin(), v10.end(), 0.0)/v10.size();
   v10.clear();
-  return pm10_avarage; 
+  return pm10_average;
 }
 
-void avarageLoop(){
+void averageLoop(){
   if (v25.size() >= stime){
-    apm25 = getPM25Avarage();  // global var for display
-    apm10 = getPM10Avarage();
+    apm25 = getPM25Average();  // global var for display
+    apm10 = getPM10Average();
+
+////////////
```

Comment:
The am2320 sensor read lines and print shouldn't be included in `avarageLoop()` method, these lines should be a independent method, like `getHumidityRead()` for example, and this method should called from main loop like `wifiLoop()` for example, with the same validation `v25.size()==0` for send it on each `stime` reached.

Reply:
Done, two new routines:
getHumidityRead()
humidityLoop()